packages/editor/src/components/url-input/index.js

```diff
@@ -159,22 +169,33 @@ class URLInput extends Component {
 				} );
 				break;
 			}
+			case TAB:
 			case ENTER: {
 				if ( this.state.selectedSuggestion !== null ) {
 					event.stopPropagation();
 					const post = this.state.posts[ this.state.selectedSuggestion ];
-					this.selectLink( post );
+					this.selectLink( post, event );
 				}
+				break;
 			}
 		}
 	}
 
-	selectLink( post ) {
+	selectLink( post, event ) {
 		this.props.onChange( post.url, post );
 		this.setState( {
 			selectedSuggestion: null,
 			showSuggestions: false,
 		} );
+
+		// Announce a link has been selected when tabbing away from the input field.
+		if ( event.keyCode === TAB ) {
```

Comment:
Why should we only announce this on "tab" only? I prefer to do it consistently if possible.

Reply:
Because when pressing Enter, the form is submitted and the link gets _inserted_. There's another speak message for that, see  "Link inserted" in `submitLink()` in `LinkContainer`.

Reply:
Gotcha, instead of passing the event object here, I'd personally just move this to the switch/case above